app/Models/User.php

```diff
@@ -114,6 +114,10 @@ public function fullName()
         return "{$this->first_name} {$this->last_name}";
     }
 
+    public function getFullNameAttribute()
```

Comment:
It could return fullName in that method rather than duplicating code, but the getFullNameAttribute creates a virtual full_name property on the model to be used in list().